_maps/configs/sugarcube.json

```
@@ -0,0 +1,11 @@
+{
+	"map_name": "Sugarcube-Class Prison Bus Ship",
+	"map_short_name": "Sugarcube-class",
+	"map_path": "_maps/shuttles/shiptest/sugarcube.dmm",
+	"map_id": "sugarcube",
+	"limit": 2,
+	"job_slots": {
+		"Prisoner": 6
```

```suggestion
		"Prisoner": 4
```
6 is still just a bit too much for a ship this small; if they want more crew they can open more slots